.travis.yml

```diff
@@ -1,20 +1,18 @@
-sudo: false
+language: elm
 
-language: node_js
-node_js: node
+elm: latest-0.19.0
 
-cache:
-  directories:
-    - node_modules
-
-os:
-  - linux
-
-env: ELM_VERSION=0.19.0
-
-install:
-  - npm install
+elm-test: latest-0.19.0
+elm-format: latest-0.19.0
 
 script:
-  - npm test
-  - npm run approve
+  - npm run test
+  - npm run build-examples
+  - ./examples/elm-test
+  - ./examples/git init
+  - ./examples/graphqelm https://example.com
+  - ./examples/simple --name Steve
+  - ./examples/validation --name Steve
+  - ./examples/curl https://example.com
+  - echo "a\nb\nc" | ./examples/grep  b
```

Comment:
These scripts (that actually run the examples and make sure they exit with a code of 0) are new in this PR. Should they be moved into package.json? The only thing is it becomes a very long line with lots of `&&`s if I do that.

Reply:
Yeah, I think it will make it more maintainable to have them in a script, and then call that script from travis (rather than having it inline in the .travis.yml).

But a bash script would work, too! Then you don't need to squeeze everything onto one line. You can use `set -e` in the bash script to have the script exit if any of the lines in the script return non-0 exit status: https://stackoverflow.com/a/2871034.

Reply:
Agree!